src/section/section.tsx

```diff
@@ -67,15 +81,24 @@ export const Section = ({
             </GridColumn>
           )}
 
-          {illustration === "snakeLongNeck" && (
-            <GridColumn
-              colSpan={2}
-              mdColSpan={2}
-              className="mt-auto hidden md:block"
-            >
-              <SnakeLongNeck className="hidden w-full lg:block" />
-            </GridColumn>
-          )}
+         
+
+          {SX_ILLUSTRATIONS.includes(illustration as string) &&
```

Comment:
can be improve this code by moving this to an external component maybe? not a big fan of the inline function call in the JSX

Reply:
put in an external component